source/manual/github-unavailable.html.md

```diff
@@ -6,62 +6,53 @@ layout: manual_layout
-
-We can deploy from AWS CodeCommit instead of GitHub, if GitHub is down or if we need a private place to develop fixes for security vulnerabilities before they are deployed.
-
-To do so, trigger the [`Deploy_App`][Deploy_App] Jenkins job, but check the `DEPLOY_FROM_AWS_CODECOMMIT` box to deploy from AWS CodeCommit.
+Setup your local environment for [checking out code from CodeCommit](/manual/howto-checkout-and-commit-to-codecommit.html)
 
-### Making changes to code in AWS CodeCommit before deployment
+## Drill creating and deploying a branch from CodeCommit
 
-If you need to deploy from AWS CodeCommit because GitHub is down or you need to test a change of a sensitive nature, you will likely need to push a new branch of code to CodeCommit.
+If GitHub.com is down, old images can still be deployed manually in Argo.
 
-Setup your local environment for [checking out code from CodeCommit](/manual/howto-checkout-and-commit-to-codecommit.html)
+1. Follow the steps for [checking out and commiting from CodeCommit](/manual/howto-checkout-and-commit-to-codecommit.html) to make a change.
+1. Ensure the following instructions are all run from the repository root directory.
+1. Get the IMAGE_TAG for your change as you will need to push this to the ECR manually in order to deploy the change from Argo.
 
-### Deploying a hotfix from AWS CodeCommit
+```
+LOCAL_HEAD_SHA="$(git rev-parse HEAD)"
+IMAGE_TAG="release-${LOCAL_HEAD_SHA}"
+ECR_REGISTRY="172025368201.dkr.ecr.eu-west-1.amazonaws.com"
+REPO=$(awk -F '=' /GOVUK_APP_NAME/'{print $2}' Dockerfile)
+```
 
-Once you've [pushed your hotfix branch to CodeCommit](#making-changes-to-code-in-aws-codecommit-before-deployment), you should:
+1. Build the docker image locally with the tag named after the latest sha.
 
-1. Create a pull request in the CodeCommit UI.
-1. Ask another developer to review it as a sense check.
-1. With the branch reviewed, you can then [deploy it](#deploy-from-aws-codecommit), specifying your branch name as the `TAG` (in the example above, 'hotfix'). Do this in Integration or Staging first, then manually test the changes.
-  - In the example above, you can check the README was removed by SSH'ing into the relevant machine and executing `ls /var/apps/{REPO_NAME}` to confirm that the README is absent.
-1. Once you're confident in your changes, deploy the branch to Production.
-1. Record the missing deployment in the Release app.
+```
+ docker build --platform linux/amd64 -f Dockerfile -t $IMAGE_TAG .
+```
 
-### Tidy up after deploying a hotfix
+1. Push the image to ECR after logging in.
```

Comment:
Clearer wording, otherwise it sounds like you need to log in before running the step below:

```suggestion
1. Log in and push the image to ECR:
```